grammar.js

```diff
@@ -124,7 +124,8 @@ module.exports = grammar(C, {
       ':',
       commaSep1(seq(
         optional(choice('public', 'private', 'protected')),
-        $._class_name
+        $._class_name,
+        optional('...')
```

Comment:
For class base clauses (and the field initializers below), I just added an optional `...` token instead of making a new `variadic_base_class_clause` and `variadic_field_initializer`, but if you think the distinction would be useful, I can rework this to add them as separate rules.